docs/topics/mtr-rn-new-features-1-2-0.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-1-2-0_{context}"]
+= New features
+
+This section describes the new features of the {ProductName} ({ProductShortName}) 1.2.0.
+
+. Decompile and analyze Java 17 based applications
```

Comment:
@HagayVider1 
I think this bullet should be a noun(s) for the sake of uniformity, because all other bullets are (enhancements, compatibility, upgrade). I would suggest "Decompilation and analysis of applications based on Java 17".